suricata.yaml.in

```diff
@@ -312,6 +312,13 @@ outputs:
         - ssh
         - mqtt:
             # passwords: yes           # enable output of passwords
+            # msg-log-limit: 1kb       # limit size of logged messages in bytes.
```

Comment:
Not only messages, also topics and all fields, right ?

Reply:
I'll adjust the wording, yeah.

Reply:
BTW I also added limiting for all strings that may appear in MQTT 5 properties.

Reply:
Cool